plugin/r/src/main/java/com/twosigma/beaker/r/utils/RServerEvaluator.java

```diff
@@ -508,9 +488,16 @@ public void run() {
             // direct graphical output
             String tryCode;
             connection.eval("do.call(svg,c(list('" + file + "'), beaker::saved_svg_options))");
-            tryCode = "beaker_eval_=withVisible(try({ " + j.codeToBeExecuted + "\n},silent=TRUE))\n"+
-                    "list(beaker_eval_, beaker:::convertToJSON(beaker_eval_$value, beaker:::collapse_unit_vectors))";
+            tryCode = "warn <- {};\n"
```

Comment:
instead of using variables named "warn" and "err",
use "beaker_warn_" and "beaker_err_" to avoid any conflicts with user code.